chat-ui/src/DevCenterChatbot.tsx

```diff
@@ -526,26 +564,70 @@ const Message = ({
           : messageData.content}
       </LGMessage>
       {displaySuggestedPrompts && (
-        <div className={styles.message_prompts}>
-          <MessagePrompts label="Suggested Prompts">
-            {suggestedPrompts.map((sp, idx) => (
+        <MessagePrompts
+          messagePrompts={suggestedPrompts}
+          messagePromptsOnClick={suggestedPromptOnClick}
+        />
+      )}
+    </Fragment>
+  );
+};
+
+type MessagePromptsProps = {
+  messagePrompts: string[];
+  messagePromptsOnClick: (prompt: string) => void;
+};
+
+const MessagePrompts = ({
+  messagePrompts,
+  messagePromptsOnClick,
+}: MessagePromptsProps) => {
+  const [inProp, setInProp] = useState(true);
+  const [suggestedPromptIdx, setSuggestedPromptIdx] = useState(-1);
+  const nodeRef = useRef(null);
+
+  const duration = 300;
+
+  const defaultStyle = {
+    transition: `opacity ${duration}ms ease-in`,
+    opacity: 0,
+  };
+
+  const transitionStyles = {
+    entering: { opacity: 1 },
+    entered: { opacity: 1 },
+    exiting: { opacity: 0 },
+    exited: { opacity: 0 },
+    unmounted: { opacity: 0 },
+  };
```

Comment:
Let's use emotion for this instead of the jsx `style` prop. Also at this point we're not actually using any of these states so RTG isn't adding anything until we wire it up.

Since we're only doing CSS transitions right now I think it'd be easier to switch to [`CSSTransition`](https://reactcommunity.org/react-transition-group/css-transition) - then we don't have to manually process the state. We just right the CSS we want for each state and it's wired up automatically.

You should be able to put all this in the `message_prompts` css (note the different state names compared to regular `Transition`):

```ts
  message_prompts: css`
    margin-left: 70px;
    @media screen and (max-width: 804px) {
      margin-left: 50px;
    }

    transition: opacity ${duration}ms ease-in;

    &-enter {
      opacity: 0;
    }
    &-enter-active {
      opacity: 1;
    }
    &-exit {
      opacity: 1;
    }
    &-exit-active {
      opacity: 0;
    }
  `,
```

then you'll replace `Transition` with `CSSTransition`, add the prop `classNames={styles.message_prompts}`, and then add `ref={nodeRef}` to the div that has the `message_prompts` style.